LayoutTests/fast/events/touch/ios/programmatic-scrolling-on-touchmove.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ useFlexibleViewport=true ] -->
+<html>
+<meta name="viewport" content="width=device-width">
+<head>
+<script src="../../../../resources/js-test.js"></script>
+<script src="../../../../resources/ui-helper.js"></script>
+<style>
+html, body {
+    margin: 0;
+    text-align: center;
+    width: 100%;
+    font-family: system-ui;
+}
+
+.container {
+    background: linear-gradient(to bottom, red 0%, green 50%, blue 100%);
+    width: 100%;
+    height: 3000px;
+    display: inline-block;
+}
+</style>
+<script>
+jsTestIsAsync = true;
+
+addEventListener("load", async () => {
+    description("To manually run this test, swipe up to scroll down; scrolling should not be jittery.");
+    let scrollStartTimerID = null;
+    let startPosition = 0;
+    let scrollDeltas = [];
+    const container = document.querySelector(".container");
+    container.addEventListener("touchstart", (event) => {
+        startPosition = event.touches[0].pageY;
+        event.preventDefault();
+    });
+
+    container.addEventListener("touchmove", (event) => {
+        const position = Math.round(event.touches[0].pageY);
+        const delta = Math.max(position, startPosition) - Math.min(position, startPosition)
```

Comment:
nit: `Math.abs(position - startPosition)`?

Reply:
Ah, I forgot to remove this `delta` altogether since it's now unused. Will do that…